stripe/src/main/java/com/stripe/android/Logger.kt

```diff
@@ -8,6 +8,14 @@ internal interface Logger {
     fun info(msg: String)
 
     companion object {
+        internal fun get(enableLogging: Boolean): Logger {
```

Comment:
I think `get` is a bit unclear as it doesnt explicitly say whether it returns a new instance or an existing one? maybe rename it to something like `instance` for clarity?

Reply:
I renamed to `getInstance()`